benchmark/composer.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "mongodb/mongodb-benchmark",
```

Comment:
This is a proposal to define different dependencies for the benchmark: create a dedicated composer project. There is the PHP version and now the amphp dependency (which requires PHP 8).

The library `mongodb/mongodb` is installed from the parent path.